core/client/fs/src/main/java/alluxio/client/file/cache/store/MemoryPageStore.java

```diff
@@ -35,7 +35,8 @@ public class MemoryPageStore implements PageStore {
   private ConcurrentHashMap<PageId, byte[]> mPageStoreMap = new ConcurrentHashMap<>();
 
   @Override
-  public void put(PageId pageId, byte[] page) throws IOException {
+  public void put(PageId pageId, byte[] page, boolean isTemporary) throws IOException {
+    //TODO(beinan): support temp page for memory page store
```

Comment:
when will this be done?

Reply:
I think very soon, actually I'm just working on the temp page of rocksdb.